### PR TITLE
Shred homegrown HTTP client in Amazon ingester and replace with Apach…

### PR DESCRIPTION
**Pull request by emilssolmanis**
…e HC

### CODE REVIEW

**Comment by dragospitica** [comment]
LGTM

File: src/main/java/org/atlasapi/remotesite/amazonunbox/AmazonUnboxHttpFeedSupplier.java [hunk 1]
```diff
@@ -31,43 +33,43 @@
...
     @Override
     public ImmutableList<AmazonUnboxItem> get() {
-        
-        try {
+        HttpGet get = new HttpGet(uri);
+        final ItemCollatingAmazonUnboxProcessor processor = new ItemCollatingAmazonUnboxProcessor();
+        final AmazonUnboxContentHandler handler = new AmazonUnboxContentHandler(processor);
+
+        try (
+                CloseableHttpClient client = HttpClients.createDefault();
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/amazonunbox/AmazonUnboxHttpFeedSupplier.java]: Do we need to make a new one every time?

Comment (emilssolmanis) [inline src/main/java/org/atlasapi/remotesite/amazonunbox/AmazonUnboxHttpFeedSupplier.java]: Depends. Given the job only runs ever 12 hours, don't see a point in leaving it open consuming threads and whatnot.

Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/amazonunbox/AmazonUnboxHttpFeedSupplier.java]: :+1:

**Pull request merged**